File: WikiCodeParser/Models/WikiRevisionBook.cs
namespace WikiCodeParser.Models
{
    public class WikiRevisionBook
    {
        public int ID { get; set; }
        public int RevisionID { get; set; }
        public string BookName { get; set; }
        public string ChapterName { get; set; }
        public int ChapterNumber { get; set; }
        public int PageNumber { get; set; }
    }
}

File: php/src/Models/WikiRevisionBook.php
<?php

namespace LogicAndTrick\WikiCodeParser\Models;

class WikiRevisionBook
{
    public ?int $id;
    public ?int $revisionID;
    public ?string $bookName;
    public ?string $chapterName;
    public ?int $chapterNumber;
    public ?int $pageNumber;
}